src/main/java/teammates/ui/template/FeedbackQuestionFeedbackPathSettings.java

```diff
@@ -48,4 +35,27 @@ public void setNumOfEntitiesToGiveFeedbackToValue(int numOfEntitiesToGiveFeedbac
         this.numOfEntitiesToGiveFeedbackToValue = numOfEntitiesToGiveFeedbackToValue;
     }
     
+    public boolean getIsCommonPathSelected() {
```

Comment:
I actually wanted the JSTL to read nicer: `.isCommonPathSelected` as opposed to `.commonPathSelected`, which can be slightly misleading